experimental/README.md

```diff
@@ -32,6 +33,13 @@ true
 
 ## Current experimental features
```

Comment:
Without a historian, I suggest just starting with 1.13, where v2 plugins, dab files, and stacks all graduated experimental (First seen in 1.12.0) @mstanleyjones